crates/worker/src/partition_processor_manager/mod.rs

```diff
@@ -85,10 +89,13 @@ pub struct PartitionProcessorManager {
     tx: mpsc::Sender<ProcessorsManagerCommand>,
 
     persisted_lsns_rx: Option<watch::Receiver<BTreeMap<PartitionId, Lsn>>>,
+    archived_lsns_channels: HashMap<PartitionId, (watch::Sender<Lsn>, watch::Receiver<Lsn>)>,
```

Comment:
This ended up a bit more involved than I expected. Since the PPM starts the tasks to create snapshots, it holds a set of channels that can update the individual PPs about successfully archived snapshots.